typescript/sdk/src/token/cosmosnativeDeploy.ts

```diff
@@ -76,6 +77,28 @@ export class CosmosNativeDeployer {
         }
       }
 
+      if (config.interchainSecurityModule) {
+        this.logger.info(`Set ISM for token`);
+
+        await this.signersMap[chain].setToken({
+          token_id: result[chain],
+          new_owner: '',
```

Comment:
why an empty string?

Reply:
not for this PR but could it be maybe worth changing the interface so that fields that are not required/we don't want to update do not need to be provided by the user?